modules/react/button/lib/deprecated_Button.tsx

```diff
@@ -25,9 +25,9 @@ export interface DeprecatedButtonProps extends GrowthBehavior {
 
 const Container = styled('button')<DeprecatedButtonProps & StyledType>(
   {
-    fontFamily: type.body.fontFamily,
-    fontSize: type.body.fontSize,
-    ...type.variant.button,
+    fontFamily: type.properties.fontFamilies.default,
+    fontSize: type.properties.fontSizes[14],
+    fontWeight: type.properties.fontWeights.bold,
```

Comment:
This should be medium because the deprecated button shouldn't be updated.

```suggestion
    fontWeight: type.properties.fontWeights.medium,
```

Reply:
Alright, that's cool. That's not what was happening though. Which is weird.